lxd/instance/drivers/driver_qemu.go

```diff
@@ -4389,6 +4430,11 @@ func (vm *qemu) LogPath() string {
 	return shared.LogPath(name)
 }
 
+// EarlyLogFilePath returns the instance's early log path.
+func (vm *qemu) EarlyLogFilePath() string {
```

Comment:
@stgraber does this need to be exported, I cant see it being used outside of the package?